app/models/store/disclosure/OtherLiabilities.scala

```diff
@@ -26,7 +26,7 @@ final case class OtherLiabilities(
   taxCreditsReceived: Option[Boolean] = None
 ) {
   def isComplete = this match {
-    case OtherLiabilities(Some(_), _, _, Some(_)) => true
+    case OtherLiabilities(Some(_), Some(_), Some(_), Some(_)) => true
```

Comment:
This breaks the completion logic. If a user doesn't select Other or Inheritance Tax then they won't be taken to those middle pages and so the section will never be complete